app/Helpers/Helpers.php

```diff
@@ -120,3 +122,42 @@ function calculatePrice($distance, $carTyeId, $provinceAirportId, $hour, $numPic
     }
 }
 
+if (!function_exists('calculateDistance')) {
+    function calculateDistance($pickupLocation, $dropoffLocation)
+    {
+        $distance = 0;
+        $numPickup = count($pickupLocation);
+        $numDropoff = count($dropoffLocation);
+        if ($numPickup > config('constance.const.one')) {
+            for ($i = 0; $i < $numPickup - config('constance.const.one'); $i++) { 
+                $origins = $pickupLocation[$i];
+                $destinations = $pickupLocation[$i++];
+                $client = new Client();
+                $response = $client->get(config('constance.google_map') . $origins . 
+                    '&destinations=' . $destinations . '&key=' . env('GOOGLE_API_KEY'));
+                $json = json_decode($response->getBody(), true);
+                $distance += (int)$json['rows'][0]['elements'][0]['distance']['value'] / config('constance.const.m_to_km');
+            }
+        }
+        $origins = $pickupLocation[$numPickup - config('constance.const.one')];
+        $destinations = $dropoffLocation[config('constance.const.zero')];
+        $client = new Client();
+        $response = $client->get(config('constance.google_map') . $origins . 
+            '&destinations=' . $destinations . '&key=' . env('GOOGLE_API_KEY'));
+        $json = json_decode($response->getBody(), true);
+        $distance += (int)$json['rows'][0]['elements'][0]['distance']['value'] / config('constance.const.m_to_km');
```

Comment:
```suggestion
        $distance += (int) $json['rows'][0]['elements'][0]['distance']['value'] / config('constance.const.m_to_km');
```
sửa các chỗ tương tự

Reply:
vâng ạ